My Series List/My Series List/View/DetailsView/DetailsViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DetailsViewController.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 18/11/2021.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController, DetailsViewDelegate {
```

Comment:
This class should be named DetailsController. There is no need for suffix View on classes names